packages/web/src/pages/upload-page/components/EditPageNew.tsx

```diff
@@ -78,35 +89,88 @@ export const EditPageNew = (props: EditPageProps) => {
     [onContinue, setTracks, tracks]
   )
 
-  const isMultiTrack = tracks.length > 1
-
   return (
     <Formik<TrackEditFormValues>
       initialValues={initialValues}
       onSubmit={onSubmit}
       validationSchema={EditTrackSchema}
     >
-      {() => (
-        <Form>
-          <div className={cn(layoutStyles.row, layoutStyles.gap2)}>
-            <div className={styles.editForm}>
-              <TrackMetadataFields playing={false} />
-              <TrackModalArray />
-              <PreviewButton playing={false} onClick={() => {}} />
-            </div>
-            {isMultiTrack ? <MultiTrackSidebar tracks={tracks} /> : null}
-          </div>
-          <div className={styles.continue}>
-            <HarmonyButton
-              variant={HarmonyButtonType.PRIMARY}
-              text='Continue'
-              name='continue'
-              iconRight={IconArrow}
-              className={styles.continueButton}
-            />
-          </div>
-        </Form>
-      )}
+      {TrackEditForm}
     </Formik>
   )
 }
+
+const TrackEditForm = (props: FormikProps<TrackEditFormValues>) => {
+  const { values } = props
+  const isMultiTrack = values.trackMetadatas.length > 1
+
+  return (
+    <Form>
+      <div className={cn(layoutStyles.row, layoutStyles.gap2)}>
+        <div className={styles.formContainer}>
+          {isMultiTrack ? <MultiTrackHeader /> : null}
+          <div className={styles.trackEditForm}>
+            <TrackMetadataFields playing={false} />
+            <TrackModalArray />
+            <PreviewButton playing={false} onClick={() => {}} />
+          </div>
+          {isMultiTrack ? <MultiTrackFooter /> : null}
+        </div>
+        {isMultiTrack ? <MultiTrackSidebar /> : null}
+      </div>
+      <div className={styles.continue}>
```

Comment:
so this should disappear for multitrack i think.
<img width="810" alt="Pasted Graphic 1" src="https://github.com/AudiusProject/audius-client/assets/8230000/4530d224-b1d2-4b0b-8334-d02bd18a6bcd">

Reply:
oh yeah this is also not the final location for the button so I didn't do that, but I'll add the conditional just so we remember to keep it later